components/profile/ConnectWithFacebook.tsx

```
@@ -0,0 +1,118 @@
+            fbid: page.id || "",
+            category: page.category || "",
+            accessToken,
+            category_list: page.category_list || [],
+            tasks: page.tasks || [],
+            instagram_business_account: page.instagram_business_account || "",
+          };
+
+          const pageDocRef = doc(socialsRef, page.id);
+
+          await setDoc(pageDocRef, pageData);
+        });
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)